### PR TITLE
Fix issues in LIP 0027

### PR DESCRIPTION
**Pull request by AndreasKendziorra**
- There are two nested lists in LIP 0027 that are not correctly rendered on GitHub due to the special rule of requiring 4 spaces for indentation (at least for numbered lists). This is fixed here.
- One example of an invalid _Lisk JSON schema_ is fixed.
- Make clear that additional bytes make a binary message invalid.
- Make clear that only valid binary messages can be decoded.
- Remove redundant statement regarding decoding unknown fields as binary messages with unknown fields are invalid.

### CODE REVIEW

File: proposals/lip-0027.md [hunk 1]
```diff
@@ -485,13 +485,10 @@ A binary message is valid, with respect to the corresponding data and schema, if
...
 3. The binary message contains exactly one value for each property specified in the schema.
-
-  There are three exceptions to this rule:
-
-  1. Empty _Arrays_ do not appear in the binary message.
-  2. [Non-packed arrays](#array-of-string-object-and-bytes) may contain multiple values.
-  3. Non-required properties for which no value was provided do not appear in the binary message. In particular, the key is also NOT contained in the binary message.
-
+    There are three exceptions to this rule:
+    1. Empty _Arrays_ do not appear in the binary message.
```
[the remark this answers is not in the document]
Comment (AndreasKendziorra) [reply proposals/lip-0027.md]: fixed

**Review by IkerAlus — APPROVED** [review]
lgtm

**Review by ricott1 — APPROVED** [review]

**Review by MaximeGagnebin — APPROVED** [review]

**Review by IkerAlus — APPROVED** [review]

**Review by MaximeGagnebin — APPROVED** [review]

File: proposals/lip-0027.md [hunk 2]
```diff
@@ -84,9 +84,9 @@ To solve the above shortcomings of JSON schema validation, we propose to introdu
...
+The encoding specification of this proposal is based on [Protocol Buffers](https://developers.google.com/protocol-buffers), a widely used language-neutral serialization method. The encoding and decoding rules were chosen in such a way that any decoder following the protobuf specification with the correct .proto file decodes valid binary messages correctly. Hence, third party implementations could use other protobuf implementations to decode binary messages derived from this protocol. [Appendix B](#appendix-b-json-schema-to-protobuf) describes how to create a .proto file corresponding to a Lisk JSON schema.
```
Comment (ricott1) [inline proposals/lip-0027.md:87]:
```suggestion
The encoding specification of this proposal is based on [Protocol Buffers](https://developers.google.com/protocol-buffers), a widely used language-neutral serialization method. The encoding and decoding rules were chosen in such a way that any decoder following the protobuf specification with the correct `.proto` file decodes valid binary messages correctly. Hence, third party implementations could use other protobuf implementations to decode binary messages derived from this protocol. [Appendix B](#appendix-b-json-schema-to-protobuf) describes how to create a .proto file corresponding to a Lisk JSON schema.
```

Comment (ricott1) [reply proposals/lip-0027.md:87]: Or maybe normal quotes, just to distinguish the file name. Notice that there are several occurrences I didn't highlight

**Review by ricott1 — APPROVED** [review]
A minor comment you can choose to disregard

File: proposals/lip-0027.md (hunk 2 above)
Comment (AndreasKendziorra) [reply proposals/lip-0027.md]: Change here and in all other occurrences.

**Review by shuse2 — APPROVED** [review]

**Review by MaximeGagnebin — APPROVED** [review]
Thanks for the changes, LGTM.

**Pull request merged**